internal/runner/objectstore.go

```diff
@@ -479,6 +480,7 @@ func (s *objStore) Fetch(ctx context.Context, name string, unpack bool, output s
 		case s.ErrorC <- err:
 		default:
 		}
+		fmt.Println(spew.Sdump(err), "stack", stack.Trace().TrimRuntime())
 		// If we had a working file get rid of it, this is because leaving it in place will
```

Comment:
What does spew.Sdump() do? We are adding extra dependency in our code on some (personal?) repo AFAIU.
Is it worth it?

Reply:
Yes it has been worth it so far for debugging like output where I want to watch something for a couple of releases while it is deployed.  Go has improved in this area with the format specifier "%+v" but it is still not as useful because it does not have a nice json format which I can used with the observability/wide events side of things.  I am watching the Go releases to see if this comes into the language a bit more.  There have been libraries that over time I have been able to get rid of but it has been slow sometimes to replace utility packages like spew.